molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisUi.java

```diff
@@ -13,4 +17,6 @@
 	 * @return menu or null if menu does not exist or is not accessible to user
 	 */
 	MolgenisUiMenu getMenu(String menuId);
+
+	String getMenuJson();
```

Comment:
this seems a bit strange, maybe we should have an jsonMolgenisUi that implements MolgenisUi and returns json when requesting the menu